website/static/js/folderPickerNodeConfig.js

```diff
@@ -80,61 +80,62 @@ var FolderPickerViewModel = oop.defclass({
 
         self.messages = {
             invalidCredOwner: ko.pureComputed(function() {
-                return 'Could not retrieve ' + self.addonName + ' settings at ' +
-                    'this time. The credentials associated with this ' + self.addonName + ' account may no longer be valid.' +
-                    ' Try disconnecting and reconnecting the ' + self.addonName + ' account on your <a href="' +
+                var addonName = $osf.htmlEscape(self.addonName);
+                return 'Could not retrieve ' + $osf.htmlEscape(addonName) + ' settings at ' +
+                    'this time. The credentials associated with this ' + $osf.htmlEscape(addonName) + ' account may no longer be valid.' +
+                    ' Try disconnecting and reconnecting the ' + $osf.htmlEscape(addonName) + ' account on your <a href="' +
                     self.urls().settings + '">account settings page</a>.';
             }),
             invalidCredNotOwner: ko.pureComputed(function() {
-                return 'Could not retrieve ' + self.addonName + ' settings at ' +
-                    'this time. The ' + self.addonName + ' addon credentials may no longer be valid.' +
-                    ' Contact ' + self.ownerName() + ' to verify.';
+                return 'Could not retrieve ' + $osf.htmlEscape(addonName) + ' settings at ' +
```

Comment:
invalidCreatorOwner has an addonName variable, but invalidCredNotOwner does not, so you probably have to add that. Or JS scope rules are just terrible, but I can't remember, so do whatever is needed here.